numpy/__init__.pyi

```diff
@@ -3313,6 +3327,21 @@ class floating(inexact[_NBit1]):
         __args: Union[L[0], Tuple[()], Tuple[L[0]]] = ...,
     ) -> float: ...
     def tolist(self) -> float: ...
+    def is_integer(self: float64) -> bool: ...
```

Comment:
A number of these methods are only available to `np.float64` (xref https://github.com/numpy/numpy/issues/13375).